docs/quickstart.md

```diff
@@ -1,11 +1,11 @@
 Quick Start
 ===========
 
-circe is published to [Maven Central][maven-central] and cross-built for Scala 2.12 and 2.13,
+circe is published to [Maven Central][maven-central] and cross-built for Scala 2.12, 2.13 and 3.x,
 so you can just add the following to your build:
 
 ```scala
-val circeVersion = "0.14.1"
+val circeVersion = "0.14.10"
```

Comment:
perhaps we should use a banner her and refer to that instead.

`![Maven Central](https://img.shields.io/maven-central/v/io.circe/circe-core_2.13?versionPrefix=0.14)`

then we don't have to main the version on release

Reply:
hmm I wouldn't, you want dependency strings that are accessible I think. But we could source this info from sbt, if this is mdoc - I'll look into it

Reply:
done

Reply:
and I've confirmed that it does use 0.14.10 at the moment.